templates/default/rt/rtqueuelist.html

```diff
@@ -52,7 +52,15 @@ <H1>{$layout.pagetitle}</H1>
 	{foreach from=$queues item=queue}
 	<TR class="highlight {cycle} lms-ui-rt{if $queue.deleted == 1 && $queue.deltime != 0}-deleted{elseif $queue.delcount != 0 && (ConfigHelper::checkPrivilege('helpdesk_advanced_operations'))}-partly-deleted{else}{/if}" onClick="return self.location.href='?m=rtqueueview&amp;id={$queue.id}';">
 		<TD class="nobr bold">
-			{if $queue.unread}<img src="img/ring-blue.png" alt="">{else}<IMG src="img/queue.gif" ALT="">{/if}
+		{if $queue.critical != 0}
```

Comment:
Nie wiem czy takie ładne formatowanie kodu nie spowoduje z drugiej strony złego wyświetlania całości w różnych przeglądarkach.

Reply:
Wrzuciłem stałe zamiast cyfr. Zaczyna się robić mało czytelnie dlatego rozrzuciłem to. Sprawdziłem na chrome i ff - działa ok.